arkane/qchem.py

```diff
@@ -273,12 +273,13 @@ def loadEnergy(self, frequencyScaleFactor=1.):
         """
         E0 = None
         with open(self.path, 'r') as f:
-            for line in f:
+            lines = f.readlines()
+            for line in lines:
```

Comment:
The original implementation is preferable because it treats the file as an iterator, so only one line is loaded at a time. If you use readlines(), then the entire file has to be loaded into memory.

Reply:
OK, I think this isn't related to this PR (sorry) so I removed it, we could discuss it at another time (I think it might be necessary for running Arkane through ARC). Thanks for bringing this up, I removed the commit.